doc/rules/basic/single_line_empty_body.rst

```diff
@@ -0,0 +1,23 @@
+===============================
+Rule ``single_line_empty_body``
+===============================
+
+Empty body of class or function must be abbreviated as ``{}`` and placed on the
+same line as the previous symbol, separated by a space.
+
+Examples
+--------
+
+Example #1
+~~~~~~~~~~
+
+.. code-block:: diff
+
+   --- Original
+   +++ New
+    <?php function foo(
```

Comment:
PSR-12 standard requires:
- for non-anonymous function/class: line break before `{` and one before `}`, this is common coding standard and it helps to reduce diff noise
- for anonymus function/class no line break

So the rule should be configureable for these usecases.

Reply:
PSR-12 is replaced by [PER-CS](https://www.php-fig.org/per/coding-style/) which enforces/encourages `{}` for both scenarios, so there's no point in supporting outdated coding standard.